tests/modeling/test_bayesianblocks.py

```diff
@@ -0,0 +1,23 @@
+import os
+import numpy as np
+import statutils
```

Comment:
Can you replace `statutils` with `scikit-stats` here ?

Reply:
Note that the module will probably be `skstats` as per the usual "transformation, say `scikit-learn` to `sklearn`, etc.

Reply:
True.